File: android/src/main/kotlin/com/vferreirati/flutter_stone/TransactionEvent.kt
package com.vferreirati.flutter_stone

data class TransactionEvent(
        val errorCode: Int? = null,
        val transactionDate: String? = null,
        val cardBrand: String? = null,
        val cardNumber: String? = null,
        val cardHolderName: String? = null,
        val transactionNumber: String? = null,
        val typeOfTransaction: Int? = null,
        val authorizationCode: String? = null,
        val instalmentCount: Int? = null,
        val transactionStatus: Int? = null
)

File: lib/src/models/transaction_event.dart
part of flutter_stone;

@JsonSerializable()
class TransactionEvent {
  final int errorCode;
  final String transactionDate;
  final String cardBrand;
  final String cardNumber;
  final String cardHolderName;
  final String transactionNumber;
  final int typeOfTransaction;
  final String authorizationCode;
  final int instalmentCount;
  final int transactionStatus;

  TransactionEvent(this.errorCode, this.transactionDate, this.cardBrand, this.cardNumber, this.cardHolderName, this.transactionNumber, this.typeOfTransaction, this.authorizationCode, this.instalmentCount, this.transactionStatus);

  static TransactionEvent fromJson(Map<String, dynamic> json) => _$TransactionEventFromJson(json);
  Map<String, dynamic> toJson() => _$TransactionEventToJson(this);
}